contracts/src/systems/raid.cairo

```diff
@@ -0,0 +1,128 @@
+#[system]
+mod Raid {
```

Comment:
thinking of calling this AttackTravelingEntity rather than raiding

Reply:
We could but i think that'll be limiting. With what we have here, we could basically just add the `Health` component to anything and it becomes raidable. Whether travelling or not. It doesn't actually have to be movable. If it isn't movable, its live_location would just be its `Position`